diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a49318aa..bc83cab53 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -181,7 +181,7 @@ macro(yuzu_find_packages)
             if (FN_FORCE_REQUIRED)
                 find_package(${PACKAGE_PREFIX} ${PACKAGE_VERSION} REQUIRED)
             else()
-                find_package(${PACKAGE_PREFIX} ${PACKAGE_VERSION})
+                find_package(${PACKAGE_PREFIX} ${PACKAGE_VERSION} REQUIRED)
             endif()
         endif()
         if (NOT ${PACKAGE_PREFIX}_FOUND)
@@ -539,7 +539,6 @@ else()
     set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT yuzu-cmd)
 endif()
 
-
 # Installation instructions
 # =========================
 
@@ -549,9 +548,12 @@ endif()
 # http://standards.freedesktop.org/shared-mime-info-spec/shared-mime-info-spec-latest.html
 if(ENABLE_QT AND UNIX AND NOT APPLE)
     install(FILES "${PROJECT_SOURCE_DIR}/dist/yuzu.desktop"
-            DESTINATION "${CMAKE_INSTALL_PREFIX}/share/applications")
+            DESTINATION "${CMAKE_INSTALL_PREFIX}/share/applications"
+            COMPONENT desktop)
     install(FILES "${PROJECT_SOURCE_DIR}/dist/yuzu.svg"
-            DESTINATION "${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/apps")
+            DESTINATION "${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/apps"
+	    COMPONENT desktop)
     install(FILES "${PROJECT_SOURCE_DIR}/dist/yuzu.xml"
-            DESTINATION "${CMAKE_INSTALL_PREFIX}/share/mime/packages")
+            DESTINATION "${CMAKE_INSTALL_PREFIX}/share/mime/packages"
+            COMPONENT desktop)
 endif()
diff --git a/externals/CMakeLists.txt b/externals/CMakeLists.txt
index d1dcc403b..737556c73 100644
--- a/externals/CMakeLists.txt
+++ b/externals/CMakeLists.txt
@@ -4,10 +4,12 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/CMakeModules")
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/externals/find-modules")
 include(DownloadExternals)
 
+set(BUILD_SHARED_LIBS OFF)
+
 # xbyak
 if (ARCHITECTURE_x86 OR ARCHITECTURE_x86_64)
     add_library(xbyak INTERFACE)
-    target_include_directories(xbyak SYSTEM INTERFACE ./xbyak/xbyak)
+    target_include_directories(xbyak SYSTEM INTERFACE /usr/include/xbyak)
     target_compile_definitions(xbyak INTERFACE XBYAK_NO_OP_NAMES)
 endif()
 
@@ -93,4 +95,7 @@ if (ENABLE_WEB_SERVICE)
 endif()
 
 # Opus
-add_subdirectory(opus)
+#add_subdirectory(opus)
+add_library(opus INTERFACE)
+target_include_directories(opus SYSTEM INTERFACE /usr/include/opus)
+target_link_libraries(opus INTERFACE /usr/lib64/libopus.so)
diff --git a/src/common/CMakeLists.txt b/src/common/CMakeLists.txt
index d120c8d3d..533b1a6e4 100644
--- a/src/common/CMakeLists.txt
+++ b/src/common/CMakeLists.txt
@@ -189,7 +189,7 @@ if(ARCHITECTURE_x86_64)
 endif()
 
 create_target_directory_groups(common)
-find_package(Boost 1.71 COMPONENTS context headers REQUIRED)
+find_package(Boost 1.71 COMPONENTS context REQUIRED)
 
 target_link_libraries(common PUBLIC ${Boost_LIBRARIES} fmt::fmt microprofile)
 target_link_libraries(common PRIVATE lz4::lz4 zstd::zstd xbyak)
diff --git a/src/yuzu/CMakeLists.txt b/src/yuzu/CMakeLists.txt
index 656096c9f..a17aef8b8 100644
--- a/src/yuzu/CMakeLists.txt
+++ b/src/yuzu/CMakeLists.txt
@@ -242,7 +242,9 @@ if (YUZU_ENABLE_BOXCAT)
 endif ()
 
 if(UNIX AND NOT APPLE)
-    install(TARGETS yuzu RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin")
+    install(TARGETS yuzu 
+	    RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin"
+	    COMPONENT yuzu)
 endif()
 
 if (MSVC)
diff --git a/src/yuzu_cmd/CMakeLists.txt b/src/yuzu_cmd/CMakeLists.txt
index a15719a0f..a42b43c44 100644
--- a/src/yuzu_cmd/CMakeLists.txt
+++ b/src/yuzu_cmd/CMakeLists.txt
@@ -34,7 +34,9 @@ endif()
 target_link_libraries(yuzu-cmd PRIVATE ${PLATFORM_LIBRARIES} SDL2 Threads::Threads)
 
 if(UNIX AND NOT APPLE)
-    install(TARGETS yuzu-cmd RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin")
+    install(TARGETS yuzu-cmd 
+	    RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin"
+	    COMPONENT yuzu-cmd)
 endif()
 
 if (MSVC)
diff --git a/src/yuzu_tester/CMakeLists.txt b/src/yuzu_tester/CMakeLists.txt
index 06c2ee011..c2e89b50e 100644
--- a/src/yuzu_tester/CMakeLists.txt
+++ b/src/yuzu_tester/CMakeLists.txt
@@ -23,7 +23,9 @@ endif()
 target_link_libraries(yuzu-tester PRIVATE ${PLATFORM_LIBRARIES} SDL2 Threads::Threads)
 
 if(UNIX AND NOT APPLE)
-    install(TARGETS yuzu-tester RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin")
+    install(TARGETS yuzu-tester 
+	    RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin"
+	    COMPONENT yuzu-tester)
 endif()
 
 if (MSVC)
